rust/pact_matching/src/models/message.rs

```diff
@@ -183,7 +183,15 @@ impl Message {
                             _ => v.to_string()
                         })
                     }).collect(),
-                    _ => hashmap!{}
+                    _ => match json.get("metadata") {
```

Comment:
The `get` function returns an optional, so you can use the or functions instead of repeating the code

Reply:
aha!